package.json

```diff
@@ -10,7 +10,7 @@
     "eslint-plugin-svelte3": "^2.7.3",
     "lerna": "3.14.1",
     "prettier": "^1.19.1",
-    "prettier-plugin-svelte": "^0.7.0",
+    "prettier-plugin-svelte": "^1.4.0",
```

Comment:
On my branch (which I haven't merged in yet) I bumped this to `1.1.0` instead of `1.4.0`, only reason being that when I bumped it this high it changed a bunch of other formatting (must be some new default formatting rules).

Happy either way, just depends if you want to re-format for the newer rules?

Reply:
Yep noticed that actually - seems like it's probably best to just bump to the latest and not delay the invevitable I guess

Reply:
On this front, do we want to consider bumping prettier too? I tried bumping it as well but it changed a fair number of things, but we are quite out of date sadly.